packages/core/src/subscriptions/redux/actions/doUnsubscribeFromSubscription.js

```diff
@@ -0,0 +1,49 @@
+import * as actionTypes from '../actionTypes';
+
+/**
+ * @callback UnsubscribeFromSubscriptionThunkFactory
+ *
+ * @alias UnsubscribeFromSubscriptionThunkFactory
+ * @memberof module:subscriptions/actions
+ *
+ * @param {object} props - Props object.
```

Comment:
Same as https://github.com/Farfetch/blackout/pull/154/files#r793610815

Reply:
Updated. Can you check again?